### DISCUSSION TITLE
Mesh in Complex Geometry

### DISCUSSION THREAD

**Discussion by lalmanohar** [rmrsk/EBGeometry#57] [Q&A] [answered]
Was mesh pruning implemented in the code? I have also gone through AMRex Document but was unable to find anything.

**Comment by rmrsk** [comment]
I'm not sure which mesh you're trying to prune. If you're looking to prune surface triangles from your mesh then I recommend something like Meshlab. If you're talking about pruning volume mesh after your EB generation has gone through, then it's an AMReX thing and you need to ask the devs over there.

If you want to use STL support with AMReX, then the examples in Examples/AMReX_DCEL is what you're looking for. EBGeometry pretty much provides a "one-liner" for turning STLs into implicit functions.

**Comment by lalmanohar** [comment]


**Reply by rmrsk** [reply, marked as answer]
Ok,so none of these are STLs but are made using CSG. For Fig. 7 it's probably just a bunch of cylinders put together. Pruning covered grids is fairly simple as you just need to remove grids that fall inside the EB. But it doesn't have anything to do with EBGeometry, so I suggest you ask the AMReX team for further guidance.
